figma-shop/src/app/pages/shopping-cart/shopping-cart/shopping-cart.component.ts

```diff
@@ -53,4 +55,44 @@ export class ShoppingCartComponent implements OnInit {
     return this.countries.filter((country: string) => country.toLowerCase().includes(filterValue));
   }
 
+  public getProductFromLocal(): ProductInterface[] {
+    if (localStorage.getItem('order')) {
+      return JSON.parse(localStorage.getItem('order'));
+    }
+  }
+
+  public removeProduct(id: number): void {
+    this.orderService.removeProduct(id);
+    const productIndex = this.orderedProducts.findIndex((prod: ProductInterface) => prod.id === id);
+    this.orderedProducts.splice(productIndex, 1);
+    this.getTotal();
+  }
+
+  public addToWishList(product: ProductInterface): void {
+    alert(`${product.title} was added to wish list`);
```

Comment:
use material snack bar

Reply:
ok, in the future this line of code is redundant, it's only for now